src/vs/workbench/api/common/positron/extHostMethods.ts

```diff
@@ -281,14 +281,14 @@ export class ExtHostMethods implements extHostProtocol.ExtHostMethodsShape {
 	}
 
 	async createDocument(contents: string, languageId: string): Promise<null> {
-
 		const uri = await this.documents.createDocumentData({
-			content: contents, language: languageId
+			content: contents,
+			language: languageId,
 		});
+		const documentData = await this.documents.ensureDocumentData(uri);
```

Comment:
This is just a small cleanup to use `await` syntax that I did while I was planning to do more changes here. I didn't end up modifying this handler, but I thought we might keep the cleanup. It causes the handler to only resolve when `showTextDocument()` has resolved, which I think is probably best in terms of assumptions made by the requester.